### PR TITLE
[Circletracer] Add a function that fetches attributes by option name

### PR DESCRIPTION
**Pull request by donggillee-dev**
This is to call a function that extracts appropriate attributes according to the operator's name for visualization of the node graph.

ONE-vscode-DCO-1.0-Signed-off-by: Donggil Lee <donggillee94@gmail.com>

### CODE REVIEW

**Comment by donggillee-dev** [comment]
from draft #179
from issue #66.

**Comment by llFreetimell** [comment]
Please revise the commit msg. The msg should include "what this PR for" :)

File: src/Circlereader/util/caseOptions.ts [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    case 'UniqueOptions':
+      OptionsAttribute.getUniqueAttr(operator, modelAttribute);
+      break;
+    case 'UnpackOptions':
+      OptionsAttribute.getUnpackAttr(operator, modelAttribute);
+      break;
+    case 'WhileOptions':
+      OptionsAttribute.getWhileAttr(operator, modelAttribute);
+      break;
+  }
```
Comment (llFreetimell) [inline src/Circlereader/util/caseOptions.ts:175]: Please add `default` to handle unimplemented or unexpected options

File: src/Circlereader/util/caseOptions.ts [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Operator} from '../circle-analysis/circle/operator';
+import {NodeAttributes} from '../type/types';
+
+import {OptionsAttribute} from './optionsAttribute';
```
Comment (llFreetimell) [inline src/Circlereader/util/caseOptions.ts:20]: When I reviewed #214, 51 options are found but here are 50 options.. what is missed...?

**Comment by llFreetimell** [comment]
Title is really long.... how about
"Add a function that fetches attributes by option name"

And please update the title and comment of this PR either!

**Comment by donggillee-dev** [comment]
> Title is really long.... how about "Add a function that fetches attributes by option name"
> 
> And please update the title and comment of this PR either!

Thank you :)

File: src/Circlereader/util/caseOptions.ts (hunk 2 above)
Comment (donggillee-dev) [reply src/Circlereader/util/caseOptions.ts:20]: Oh, I forgot about `getConcatEmbeddingAttr`...T.T
I'll add it right away.

**Review by yesfordev — DISMISSED** [review]
LGTM 👍🏼

**Review by CODe1995 — APPROVED** [review]
LGTM

**Review by yesfordev — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**